React/Modules/RCTAlertManager.m

```diff
@@ -159,14 +185,22 @@ - (void)invalidate
       [alertController addAction:[UIAlertAction actionWithTitle:buttonTitle
                                                           style:buttonStyle
                                                         handler:^(__unused UIAlertAction *action) {
-        if (callback) {
-          if (allowsTextInput) {
-            callback(@[buttonKey, textField.text]);
-          } else {
-            callback(@[buttonKey]);
-          }
-        }
-      }]];
+                                                          if (callback) {
+                                                            if (allowsTextInput) {
```

Comment:
Xcode's default indenting for blocks is terrible, but please manually allign these back to something sensible.